pkg/flowser/app.go

```diff
@@ -127,7 +127,7 @@ func (a *App) appDir(installDir string) (string, error) {
 func (a *App) executable(installDir string) (string, error) {
 	files := map[string]string{
 		darwin:  "Contents/MacOS/Flowser",
-		windows: "Flowser.exe",
+		windows: "@flowserapp",
```

Comment:
We actually need to update the "app path" and not the "executable path" here: 

https://github.com/onflowser/flowser/blob/main/pkg/flowser/app.go#L123

See this example:

![image](https://user-images.githubusercontent.com/36109955/203622035-23e7a252-4cc8-44e8-b5be-acfe11f9a733.png)

"@flowserapp" is the app dir here.

Reply:
ohh that's the dir... ok will fix